### DISCUSSION TITLE
Public & private routes auth

### DISCUSSION THREAD

**Discussion by vincentcordel** [tailflow/laravel-orion#305] [Help]
I'm wondering on how to setup the following example in a controller:
- index & show should be public - unregistered users (no auth - or client credentials?)
- create, update, destroy should be private - registered users (with sanctum auth).

Also, how would I configure the corresponding endpoints routes?

**Comment by v-radev** [comment]
Hey @vincentcordel, have you tried adding the create, update and destroy methods through the auth middleware? That should be a purely Laravel configuration, is it not?
